main.js

```diff
@@ -0,0 +1,28 @@
+// Remove the "All Posts" link from the header.
+document.querySelector("header nav ul li").remove()
+
+//  Remove the span element that contains the "Read Time" in the featured post. The featured post is the first post on the page.
+let els2 = document.querySelectorAll(".featured span");
+
+for(let el2 of els2){
+    if(el2.textContent.includes("Read Time")){
+        el2.remove();
+    }
+}
```

Comment:
```suggestion
let els2 = document.querySelector(".featured span");
els2.remove()
```
Although this passes the tests, by using querySelectorAll and a for loop, you're actually removing the "Read Time" span from **every** article, not just the **featured** article as the readme.md asks.